omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -361,6 +361,31 @@
                         jq().toastmessage('showNoticeToast',"Patient has completed TB Treatment");
                 }
                                 
+                /* Prompt for malnutrition using Muac and BMI */
+                var age = jq("#age").html();
+                function BMI(ht,wt){
+                   var htInMetres = ht/100;
+                 return  wt/(htInMetres * htInMetres);
+                }
+
+                if(age &lt; 5){
+                    if(( (jq("#lastMuac").html() &gt; 0) &amp;&amp; (jq("#lastMuac").html() &lt; 11.5)) || ( (BMI(jq("#lastHeight").html(),jq("#lastWeight").html()) &gt; 0)  &amp;&amp; (BMI(jq("#lastHeight").html(),jq("#lastWeight").html()) &lt; 14)) ){
```

Comment:
@ssmusoke the reference values I have used for muac have been inspired from [captureVitals](https://github.com/METS-Programme/openmrs-module-ugandaemr-iss/blob/master/omod/src/main/webapp/resources/htmlforms/CaptureVitals.xml#L268-L288) while the BMI from a [CDC chart](https://www.calculator.net/pdf/chart-2015.pdf) while taking an average to account for both boys and girls

Reply:
@reagan-meant Where is the check for Rutf?